packages/nc-gui/pages/index/index/index.vue

```diff
@@ -174,42 +174,42 @@ const copyProjectMeta = async () => {
 
       <div class="flex-1" />
 
-      <a-dropdown v-if="isUIAllowed('projectCreate', true)" :trigger="['click']" overlay-class-name="nc-dropdown-create-project">
-        <button class="nc-new-project-menu mt-4 md:mt-0">
-          <span class="flex items-center w-full">
-            {{ $t('title.newProj') }}
-            <MdiMenuDown class="menu-icon" />
-          </span>
-        </button>
-
-        <template #overlay>
-          <a-menu class="!py-0 rounded">
-            <a-menu-item>
-              <div
-                v-e="['c:project:create:xcdb']"
-                class="nc-project-menu-item group nc-create-xc-db-project"
-                @click="navigateTo('/create')"
-              >
-                <MdiPlusOutline class="group-hover:text-accent" />
-
-                <div>{{ $t('activity.createProject') }}</div>
-              </div>
-            </a-menu-item>
-
-            <a-menu-item v-if="appInfo.connectToExternalDB">
-              <div
-                v-e="['c:project:create:extdb']"
-                class="nc-project-menu-item group nc-create-external-db-project"
-                @click="navigateTo('/create-external')"
-              >
-                <MdiDatabaseOutline class="group-hover:text-accent" />
-
-                <div v-html="$t('activity.createProjectExtended.extDB')" />
-              </div>
-            </a-menu-item>
-          </a-menu>
-        </template>
-      </a-dropdown>
+      <!--      <a-dropdown v-if="isUIAllowed('projectCreate', true)" :trigger="['click']" overlay-class-name="nc-dropdown-create-project"> -->
```

Comment:
If this UX is finalised, remove the commented code?

Reply:
@pranavxc can confirm if he wishes to retain changes.